Movie/MovieTests/CinemaListScreenViewModelTests.swift

```diff
@@ -0,0 +1,134 @@
+// CinemaListScreenViewModelTests.swift
+// Copyright © Alexandr Grigorenko. All rights reserved.
+
+@testable import Movie
+import XCTest
+
+/// Тесты CinemaListScreenViewModel
+final class CinemaListScreenViewModelTests: XCTestCase {
+    // MARK: - Private properties
+
+    private var cinemaListScreenViewModel: CinemaListScreenViewModelProtocol!
+    private var networkService: NetworkServiceProtocol!
+    private var keychainService: KeychainServiceProtocol!
+    private var coreDataService: CoreDataServiceProtocol!
+    private var imageAPIService: ImageAPIServiceProtocol!
+    private var cacheService: CacheServiceProtocol!
+    private var proxy: Proxy!
+    private var imageService: ImageServiceProtocol!
+
+    // MARK: - Public methods
+
+    override func teardown() {
+        networkService = nil
+        imageAPIService = nil
+        cacheService = nil
+        keychainService = nil
+        coreDataService = nil
+        proxy = nil
+        imageService = nil
+        cinemaListScreenViewModel = nil
+    }
+
+    override func setUp() {
```

Comment:
```suggestion
    override func setUp() {
```
super надо в тело везде добавить по проекту